lib/arc/storage/local.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Arc.Storage.Local do
+
+  def put(definition, version, {file, scope}) do
+    destination_dir = definition.storage_dir(version, {file, scope})
+    {:ok, binary} = File.read(file.path)
```

Comment:
Instead of reading the file and writing the file back to a new location, can we just copy the file directly through `File.cp`? (http://elixir-lang.org/docs/v1.0/elixir/File.html#cp/3)